pkg/resources/dnsrecord/dns.go

```diff
@@ -111,7 +111,7 @@ func desiredWildcardDNSRecord(name types.NamespacedName, dnsRecordLabels map[str
 
 	// Set the DNS management policy on the dnsrecord to "Unmanaged" if ingresscontroller has "Unmanaged" DNS policy or
 	// if the ingresscontroller domain isn't a subdomain of the cluster's base domain.
```

Comment:
Not a described part of this change, but this comment doesn't actually reflect what happens.
```
	// if the ingresscontroller domain isn't a subdomain of the cluster's base domain.
```

Reply:
Fixed by https://github.com/openshift/cluster-ingress-operator/pull/1189/commits/9ea3b247c19ca551d48adf9bb89cb787b0afd36e.